spec/requests/new_admin/notes_controller_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe NewAdmin::NotesController, type: :request do
+  describe 'GET #index' do
+    context 'when user is signed in' do
+      context 'when user is admin' do
+        let(:user) { create(:user, :admin) }
+        before { sign_in user }
+
+        context 'when no filters are applied' do
+          let!(:notes) { create_list(:note, 2) }
+
+          it 'returns status code 200 ok' do
+            get new_admin_notes_path
+
```

Comment:
Sugestão: melhoraria a legibilidade se remover essas linhas em branco no meio dos blocos 'it'.